docs/_posts/2018-06-26-jekyll-sponsoring.markdown

```diff
@@ -0,0 +1,49 @@
+---
+title: 'Jekyll and Sponsoring'
+date: 2018-06-26 16:00:00 +0200
+author: oe
+categories: [community]
+---
+
+_(TL;DR: We're open for sponsorships over our [OpenCollective page](https://opencollective.com/jekyll))_
+
+Hi Jekyllers,
+
+As you may know, Jekyll is a completely free and open source project. We offer our
+software and its related plugins and documentation at no cost because we believe
+that good software should not cost anything. We're not planning on changing that,
+but today I want to talk about a different monetary aspect of open source.
+
+Open source developers being paid for the work they do is a rare sight. Most open source
+software is effectively the result of hundreds and thousands of hours of free labor provided
+by individuals who are passionate enough to work outside of their day job to create
+software that, ironically, is being used by almost every company that offers
+digital services. It's a problem that has gotten more attention in recent years, with
+the open source community becoming more diverse and more and more companies actively
+investing in providing monetary support for open source developers.
+
+Jekyll has always been a product of volunteers. Rarely has someone been paid to implement
+a certain plugin or feature. Today, we're excited to announce that we will finally
+be able to fund our contributors! __We are opening an OpenCollective to receive
```

Comment:
Add Jekyll's open collective screenshot, once we have received our first donation ?